File: slf4j/src/main/scala/org/slf4j/impl/StaticMDCBinder.scala
package org.slf4j.impl

import org.slf4j.spi.MDCAdapter
import scribe.slf4j.ScribeMDCAdapter

class StaticMDCBinder {
  def getMDCA: MDCAdapter = ScribeMDCAdapter

  def getMDCAdapterClassStr: String = ScribeMDCAdapter.getClass.getName
}

File: slf4j/src/main/java/org/slf4j/impl/StaticMDCBinder.java
package org.slf4j.impl;

import org.slf4j.spi.MDCAdapter;
import scribe.slf4j.ScribeMDCAdapter;
import scribe.slf4j.ScribeMDCAdapter$;

public class StaticMDCBinder {
    public static final StaticMDCBinder SINGLETON = new StaticMDCBinder();

    private StaticMDCBinder() {
    }

    public MDCAdapter getMDCA() {
        return ScribeMDCAdapter$.MODULE$;
    }

    public String getMDCAdapterClassStr() {
        return ScribeMDCAdapter.class.getName();
    }
}